cmd/kubeadm/app/preflight/checks.go

```diff
@@ -191,10 +191,7 @@ func RunJoinNodeChecks() error {
 		IsRootCheck{root: true},
 		ServiceCheck{service: "docker"},
 		ServiceCheck{service: "kubelet"},
-		PortOpenCheck{port: 8080},
 		PortOpenCheck{port: 10250},
-		PortOpenCheck{port: 10251},
-		PortOpenCheck{port: 10252},
```

Comment:
This is for the node, 10251 and 10252 for cm and scheduler and 8080 for unsecured apiserver